src/core/query.ts

```diff
@@ -65,6 +65,7 @@ export interface FetchContext<
 > {
   fetchFn: () => unknown | Promise<unknown>
   fetchOptions?: FetchOptions
+  signal?: AbortSignal
```

Comment:
this looks unused - is this maybe a leftover?

Reply:
No, I had to add it because we weren't passing the signal in the `FetchContext` for the `QueryBehaviour` as I mentioned in this comment https://github.com/tannerlinsley/react-query/pull/2996#discussion_r753832455

Reply:
ah, it just looks unused, but its added via `Object.defineProperty` 👍